components/PaywallModal.vue

```diff
@@ -0,0 +1,308 @@
+            <img
+              :src="paywallBodyLogo"
+              alt="3ook Logo"
+              class="w-full max-w-[300px] sm:max-h-[200px] object-contain"
+            >
+          </div>
+        </div>
+
+        <div class="flex-1 flex flex-col justify-center items-start p-5 sm:p-12 w-full">
+          <div class="flex flex-col items-start mx-6 gap-2">
+            <div class="hidden sm:block bg-black rounded-full px-6 py-2 mb-3">
+              <span class="font-bold text-white">{{ $t('pricing_page_subscription') }}</span>
+            </div>
+            <ul class="space-y-4 text-left">
+              <li class="flex items-start">
+                <UIcon
+                  name="i-material-symbols-check"
+                  class="mt-1 mr-2 text-green-500"
+                />
+                <span>{{ $t('pricing_page_feature_1') }}</span>
+              </li>
+              <li class="flex items-start">
+                <UIcon
+                  name="i-material-symbols-check"
+                  class="mt-1 mr-2 text-green-500"
+                />
+                <span>{{ $t('pricing_page_feature_2') }}</span>
+              </li>
+              <li class="flex items-start">
+                <UIcon
+                  name="i-material-symbols-check"
+                  class="mt-1 mr-2 text-green-500"
+                />
+                <span>{{ $t('pricing_page_feature_3') }}</span>
+              </li>
+              <li class="flex items-start">
+                <UIcon
+                  name="i-material-symbols-check"
+                  class="mt-1 mr-2 text-green-500"
+                />
+                <span>{{ $t('pricing_page_feature_4') }}</span>
+              </li>
+              <li class="flex items-start">
+                <UIcon
+                  name="i-material-symbols-check"
+                  class="mt-1 mr-2 text-green-500"
+                />
+                <span>{{ $t('pricing_page_feature_5') }}</span>
+              </li>
+            </ul>
+          </div>
+
+          <!-- Price Select -->
+          <div class="flex flex-col w-full mt-12">
+            <UIcon
+              v-if="props.dismissible"
+              name="i-material-symbols-close"
+              class="
+                hidden sm:block absolute top-4 right-4
+                text-black cursor-pointer hover:text-gray-600
+                transition-colors duration-200
+              "
+              size="24"
+              @click="handleOnClose"
+            />
+            <div class="flex flex-col gap-4">
+              <!-- Yearly plan -->
+              <label
+                class="
+                  relative flex justify-between items-center
+                  border-2 rounded-2xl px-4 py-4 cursor-pointer
+                  transition-all duration-200 ease-in-out
+                  hover:shadow-lg hover:border-gray-400
+                "
+                :class="selectedPlan === 'yearly' ? 'border-black' : 'border-gray-200'"
```

Comment:
combine to one array of classes

Reply:
[🎨 Refine UI style formatting](https://github.com/likecoin/3ook-com/pull/57/commits/ebc6d70152b905cd19f5c2c4e7feedb95f77cb1e)

Reply:
[🎨 Refine UI style formatting](https://github.com/likecoin/3ook-com/pull/57/commits/ebc6d70152b905cd19f5c2c4e7feedb95f77cb1e)